### PR TITLE
Fixing basepath issue

### PR DESCRIPTION
**Pull request by nupplaphil**
FollowUp #6920 

<strike>I'm not finished yet, but I've to leave for now.</strike>

- I'd like to automatically add `system.basepath` with the determined in case there is nothing in the config file as a fallback.
- In case of an update, if both values are empty, take a default value for the config (= determined basepath)

### CODE REVIEW

**Comment by nupplaphil** [comment]
- Tested with config value in db
- Tested without config value in db

- Tested without basepath value in db and site was responding

I hope that with this and the next PR, the topic "basepath" will be fixed forever...ever..ever..er

File: src/Core/Update.php [hunk 1]
```diff
@@ -286,7 +286,7 @@ private static function updateConfigEntry(IConfigCache $configCache, ConfigFileS
 		$savedConfig = DBA::selectFirst('config', ['v'], ['cat' => $cat, 'k' => $key]);
 
 		if (!DBA::isResult($savedConfig)) {
-			return false;
+			$savedConfig = null;
```
Comment (MrPetovan) [inline src/Core/Update.php]: `$savedConfig` is expected to be an array after this.

Comment (nupplaphil) [reply src/Core/Update.php]: Ah, I was lazy and thought I can reuse the variable ;-).

**Comment by MrPetovan** [comment]
We'll have to verify this with @annando since he didn't update `pirati.ca` to `develop` yet.

**Comment by nupplaphil** [comment]
> 
> 
> We'll have to verify this with @annando since he didn't update `pirati.ca` to `develop` yet.

Crossing every finger I have ;-)

**Comment by annando** [comment]
This can last some more days, since I'm busy.

**Comment by annando** [comment]
```Parse error: syntax error, unexpected ''basepath'' (T_CONSTANT_ENCAPSED_STRING), expecting ']' in /path/to/friendica/config/local.config.php on line 96```

My config entry before this had no ```,```:

```
                'pidfile' => '/tmp/daemon.pid'
                'basepath' => '/path/to/friendica',
```

**Pull request merged**